### PR TITLE
#189 Add unpatch for dropout and consistent dropout

### PR DESCRIPTION
**Pull request by Dref360**
## Summary:

Add ability to unpatch modules to Dropout and Consistent dropout.
Not yet available for dropconnect as it is a bit more complicated.

### Features:

WIP on #189 

## Checklist:

* [x] Your code is documented (To validate this, add your module to `tests/documentation_test.py`).
* [x] Your code is tested with unit tests.
* [x] You moved your Issue to the PR state.

### CODE REVIEW

File: baal/bayesian/consistent_dropout.py [hunk 1]
```diff
@@ -152,8 +172,10 @@ def __init__(self, module: torch.nn.Module):
...
         super().__init__()
-        self.parent_module = module
-        _patch_dropout_layers(self.parent_module)
+        self.parent_module = patch_module(module)
 
     def forward(self, *args, **kwargs):
         return self.parent_module.forward(*args, **kwargs)
+
+    def unpatch(self) -> torch.nn.Module:
+        return unpatch_module(self.parent_module)
```
Comment (parmidaatg) [inline baal/bayesian/consistent_dropout.py:181]: this will make sure that `self.parent_module` is always patched but at every step the user can ask for an unpatched version right? its very useful just wanna make sure that if you call `unpatch` the internal module remains patched.

**Review by parmidaatg — APPROVED** [review]
LGTM 👍🏼

File: baal/bayesian/consistent_dropout.py (hunk 1 above)
Comment (Dref360) [reply baal/bayesian/consistent_dropout.py:181]: yes
